.github/CODEOWNERS

```diff
@@ -29,14 +29,13 @@
 /accounts/         @kjhman21
 /api/              @kjhman21
 /blockchain/       @kjhman21
-/build/            @bitaholic
-/client/           @KimKyungup
-/cmd/              @bitaholic
-/cmd/kscn/         @KimKyungup
+/build/            @bitaholic @KimKyungup @jeonghonah
+/client/           @KimKyungup @jeonghonah
+/cmd/              @bitaholic @KimKyungup @jeonghonah
```

Comment:
IMO, it would be better to be more specific than assigning many people to a top-level directory.

Reply:
We agree with the intention. However, the changes in build/node/cmd are necessary to follow-up and if necessary, the change also can be apply to the code of the service chain, so we put it in the top-level reviewer.

Reply:
During development, to align SCN/SPN/SEN with CN/PN/EN, this way is good. :-)